ludwig/encoders/text_encoders.py

```diff
@@ -353,7 +452,7 @@ def call(self, inputs, training=None, mask=None):
         else:
             hidden = transformer_outputs[0]
             hidden = reduce_sequence(hidden, self.reduce_output)
-        return hidden
+        return {'encoder_output': hidden}
 
 
 # todo tf2 check lack of TFDialoGPTModel
```

Comment:
this looks like is still in todo, right?

Reply:
Yup. Didn't look into it extensively, but still looks like HF doesn't have a TF implementation of these encoders.